### DISCUSSION TITLE
Example sites with the Webpage HTML Export (Obsidian plugin)

### DISCUSSION THREAD

**Discussion by EdwardKS** [KosmosisDire/obsidian-webpage-export#345] [Q&A] [answered]
Could you show me some websites that have static HTML content?

I'd like to see what the bi-directional linking user interface looks like!

**Comment by KosmosisDire** [comment]
You can take a look at my website if you like: https://notes.nathang.dev/

**Comment by EdwardKS** [comment]
Thank you for sharing your site!

Is there a way to see the auto-linking functions on your website?

Such as ...

- Links to this page
- Linked mentions
- Unlinked mentions
- Dataview (html on webpage)

Thank you!!

**Comment by KosmosisDire** [comment]
Hi, here is my testing site which has dataview on the plugins test page. This site is exported using active development versions so the website may break at certain times while I am working on the plugin.

https://nathang.dev/fileshare/test-vault/tests/plugins.html

Linked mentions / backlinks and unlinked mentions are not yet supported. You do have the graph view which gives some abilty to view backlinks. Backlink support is 100% coming in the future though.

**Reply by EdwardKS** [reply]
Hi Nathan,

Re: Dataview
- It's great to see it functioning so well!

Re: Backlink support
- Could you provide a short explanation of what it will accomplish?
- What is an estimated timeline?

Thanks again!

**Reply by KosmosisDire** [reply]
Hi, the goal would be for it to function very similarly to the way it does in obsidian. Basically backlinks should allow you to see a list of notes that link to this note at the bottom of the page. I am not sure what else it would accomplish, but I am open to suggestions if you are thinking of something specific.

As for the timeline, there is no estimated timeline, I will implement it when I implement it. (Unless someone submits a PR of course)

**Reply by EdwardKS** [reply]
Hi Nathan,

Re: a list of notes that link to this note at the bottom of the page
- That's great .. the same as the term Obsidian uses .. "LINKS TO THIS PAGE"
- No other suggestions .. that would be a huge plus already!!

Re: Timeline
- Is there a way I can get notified?

**Reply by KosmosisDire** [reply, marked as answer]
I can just ping you here when I implement it. But I'll try and also create an issue that you can follow. I'll let you know if I create that
